source/index.js

```diff
@@ -123,15 +123,18 @@ module.exports = async (input = 'patch', options) => {
 			{
 				title: 'Installing dependencies using Yarn',
 				enabled: () => options.yarn === true,
-				task: () => exec('yarn', ['install', '--frozen-lockfile', '--production=false']).pipe(
-					catchError(error => {
-						if (error.stderr.startsWith('error Your lockfile needs to be updated')) {
-							return throwError(new Error('yarn.lock file is outdated. Run yarn, commit the updated lockfile and try again.'));
-						}
-
-						return throwError(error);
-					})
-				)
+				task: () => {
+					return exec('yarn', ['install', '--frozen-lockfile', '--production=false']).pipe(
+						catchError(async error => {
+							if (error.stderr.startsWith('error Your lockfile needs to be updated')) {
```

Comment:
Do an early return to reduce nesting.

Reply:
I'm unsure on how to do a early return in this case, i feel return is already on top.

Reply:
```js
catchError(async error => {
	if (!error.stderr.startsWith('error Your lockfile needs to be updated')) {
		return;
	}

	if (await git.checkIfFileGitIgnored('yarn.lock')) {
		return;
	}

	throw new Error('yarn.lock file is outdated. Run yarn, commit the updated lockfile and try again.');
})
```